modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -4856,6 +4878,8 @@ private void _validateWorkflowAction(
 	@Reference
 	private ResourceLocalService _resourceLocalService;
 
+	private String _restrictedFields;
```

Comment:
As we discussed, instead of adding an attribute to the class, we should get this information in the form parameters in the methods that need it.